server/src/main/java/io/druid/segment/realtime/plumber/Sink.java

```diff
@@ -125,6 +133,11 @@ public Sink(
     makeNewCurrIndex(interval.getStartMillis(), schema);
   }
 
+  public void clearDedupCache()
```

Comment:
This method is called only by `RealTimePlumber`. When should this method be called?